config/routes.rb

```diff
@@ -29,6 +29,10 @@
     end
     resources :users, only: %i[index show new create]
 
+    resources :allocations, only: %i[index show]
+
+    resources :allocation_uplifts, only: %i[edit update create new]
```

Comment:
For a bit of extra safety, shall we nest these under members of the allocations resource, so that you can scope retrieval of the uplifts in the controller by the allocation they belong to?